README.md

```diff
@@ -174,7 +168,7 @@ We provide docker containers for Habitat, updated approximately once per year fo
 
 1. Activate the habitat conda environment: `conda init; source ~/.bashrc; source activate habitat`
 
-1. Run the testing scripts as above: `cd habitat-lab; python examples/example_pointnav.py`. This should print out an output like:
+1. Run the testing scripts as above: `cd habitat-lab; python examples/example.py`. This should print out an output like:
```

Comment:
I think the usefulness is rather limited. Since our API is exactly gym, I do not think there is that much value added in showing how to use gym (there is a ton of tutorials out there)